asf_core_data/pipeline/data_joining/merge_install_dates.py

```diff
@@ -5,136 +5,93 @@
 
 from asf_core_data.getters import data_getters
 from asf_core_data.config import base_config
+from asf_core_data.getters.epc import data_batches
 
 import numpy as np
 import pandas as pd
 
 # ---------------------------------------------------------------------------------
 
 
-def get_mcs_install_dates(epc_df, additional_features=False):
+def get_mcs_install_dates(epc_df, additional_features=True):
```

Comment:
Hey, 
Yes, I think you got it mostly right. Here just a few clarifications, I will also update the documentation accordingly.

I deduplicate the MCS data based on UPRN because it's possible that different MCS records were mapped to the same EPC property. (This would happen during the address matching, should be rare, but can happen.) Not much we can do about this here, but we can just use the first heat pump installation date as the best approximation. 

We then apply the dict to the EPC data. If there is no MCS match, the HP_INSTALL_DATE will be set to NaN for now. The other function handles the cases where there is no MCS record but there is still a heat pump installed according to EPC.

The additional feature mapping is no longer required. (Was needed for the very first analysis I did, and made it optional but probably time to remove it completely.) 

I'll update the descriptions.